.sun-ci.yml

```diff
@@ -0,0 +1,35 @@
+stages: 
+  - build
+  - test
+
+jobs: 
+  build_and_lint:
+    stage: build
+    image: node:16-alpine
+    script: 
+      - yarn install --frozen-lockfile
+      - yarn eslint .
```

Comment:
can you separate the stages

- build (preparing your backend and fe)
   - Suggestion: if you want to run one command for all, you can create a bash sript and will run both fe and be commands
- post-lint (for running linter)
- test-be 
- test-fe